code/modules/antagonists/revenant/revenant_abilities.dm

```diff
@@ -276,52 +278,59 @@
 	name = "Report this to a coder"
 	var/reveal = 80 //How long it reveals the revenant in deciseconds
 	var/stun = 20 //How long it stuns the revenant in deciseconds
-	var/locked = TRUE //If it's locked and needs to be unlocked before use
```

Comment:
Why did you reverse this variable name and function across the board? 

It's bad practice to change a variable name like this unless the naming was unclear. Please undo this.

Reply:
"Locked" is more unclear based on the perspective.
* the spell is usable when "a variable" is FALSE
* the spell is usable when "a variable" is TRUE

I am inclined to have the later one.

Reply:
I disagree. Both states are equally readable/understandable and I'm strongly against changing variable names and usage like this.

Reply:
This is really weird to me but if you say so